projects/packages/backup/changelog/revert-adding-condition

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Since this seems to have been reverting something done during this PR, it doesn't seem necessary to have a change entry for it.